### PR TITLE
Feature/add tenure api

### PR DESCRIPTION
**Pull request by urecio**

### CODE REVIEW

File: lib/api/tenure/v1/types.ts [hunk 1]
```diff
@@ -0,0 +1,116 @@
+export interface TenureDetailsProperties {
+  title: string;
+}
+
+// Once other possible param values are confirmed, we can reuse this for TenureType code
+export enum TenureTypeCode {
```
Comment (cloudratha) [inline lib/api/tenure/v1/types.ts:6]: Where is this actually used?

File: lib/api/tenure/v1/types.ts [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+
+export interface NoticeType {
+  type: string;
+  servedDate: string;
+  expiryDate: string;
+  endDate: string;
+  effectiveDate: string;
+}
+
+export interface LegacyReferences {
```
Comment (cloudratha) [inline lib/api/tenure/v1/types.ts:76]: Should this be singular?

ie 

```tsx
export interface LegacyReference {
```

File: lib/api/tenure/v1/types.ts [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+  name: string;
+  value: string;
+}
+
+export interface TenureSectionProps {
```
Comment (cloudratha) [inline lib/api/tenure/v1/types.ts:81]: Not sure if splitting this here makes sense under this context. `TenureSectionProps` is used by a component I believe.

File: webpack.config.js [hunk 4]
```diff
@@ -12,6 +16,66 @@ module.exports = (webpackConfigEnv, argv) => {
...
+    entry: {
+      common: path.join(__dirname, 'lib', 'mtfh-common.tsx'),
+      'common/lib/auth': path.join(__dirname, 'lib', 'auth'),
+      'common/lib/http': path.join(__dirname, 'lib', 'http'),
+      'common/lib/config': path.join(__dirname, 'lib', 'config'),
+      'common/lib/configuration': path.join(__dirname, 'lib', 'configuration'),
+      'common/lib/components': path.join(__dirname, 'lib', 'components'),
+      'common/lib/hooks': path.join(__dirname, 'lib', 'hooks'),
+      'common/lib/utils': path.join(__dirname, 'lib', 'utils'),
+      'common/lib/api/person/v1': path.join(
```
Comment (cloudratha) [inline webpack.config.js:28]:
```tsx
'common/lib/api/person/v1': path.join(apiPath, 'person', 'v1')
```
Might help get everything on the same line

**Review by cloudratha — CHANGES_REQUESTED** [review]

File: lib/api/tenure/v1/types.ts (hunk 1 above)
Comment (urecio) [reply lib/api/tenure/v1/types.ts:6]: Good point actually. I think nowhere

File: webpack.config.js (hunk 4 above)
Comment (urecio) [reply webpack.config.js:28]: ah nice!

**Review by cloudratha — APPROVED** [review]

**Pull request merged**